DirectEmployers/MyJobs
# [PD-3761] DSEO cache fix.

## What's this PR do?
Prevents staging configurations and production configurations from sharing the the same cache for standard facets.

Also removes a site-level cache key function to prevent me from ever needing to think about this again accidentally.

## Where should the reviewer start?
`seo/cache.py`

## How should this be manually tested?
First you have to set up cache locally, and set up a site with both a staging and production configuration. Both are pretty annoying to do.

Visit a search results page to cache the custom facets.
Toy around with the standard facets (do something like add a job while bypassing cache clearing, or add/remove facets manually in the db), but don't do anything (like saving the configuration or adding the jobs through the traditional import functions) that would change the result.
Refresh the page and view that they are the same. 
Now log in and view the page using the staging configuration. You should notice your standard facet change. 

## Any background context you want to provide?
Doing this so that staging and production configurations can use different standard facet lookup methods (facet queries vs facet tags). 


## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
